scripts/xtensa-build-all.sh

```diff
@@ -113,6 +125,7 @@ fi
 
 if [ "x$USE_PRIVATE_KEY" == "xyes" ]
 then
+	>&2 printf 'WARNING: -k and RIMAGE_PRIVATE_KEY are deprecated\n'
```

Comment:
"...use -new-switch or new ENV instead"

Reply:
To avoid duplication I added "see usage".